source/index.html

```diff
@@ -1,12 +1,183 @@
+          </li>
+          <li>
+            <a class="nav__link" href="form.html">Подбор программы</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section>
+      <h1 class="main-title">Функциональное питание для котов</h1>
+      <p class="main-tagline">Занялся собой? Займись котом!</p>
+      <img src="/img/index-can-mobile 1.png" alt="Cat Energy PRO курица.">
+      <a class="">Подобрать программу</a>
+    </section>
+    <section>
+      <div class="weight-loss">
+        <h2>Похудение</h2>
+        <p>Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim
+          поможет вашему питомцу сбросить лишний вес.</p>
+        <a href="#">Каталог Slim</a>
+      </div>
+      <div class="weight-gain">
+        <h2>Набор массы</h2>
+        <p>Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить
+          необходимые мышцы!</p>
+        <a href="#">Каталог pro</a>
+      </div>
+    </section>
+    <section class="description">
+      <h2 class="description__title">Как это работает</h2>
+      <p class="description__advantages">Функциональное питание содержит только полезные питательные вещества.</p>
+      <p class="description__advantages">Выпускается в виде порошка, который нужно лишь залить кипятком и готово.</p>
+      <p class="description__advantages">Замените один-два приема обычной еды на наше функциональное питание.</p>
+      <p class="description__advantages">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</p>
+    </section>
+    <section class="example">
+      <h2 class="example__title">Живой пример</h2>
+      <p class="example__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy
+        Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16
+        часов в день. </p>
+    </section>
+    <section class="contacts">
+      <div class="contacts__wrapper">
+        <h2 class="visually-hidden">Контакты</h2>
+        <p class="contacts__title">Приглашаем к сотрудничеству дилеров!</p>
+        <span class="contacts__adress">ул. Большая Конюшенная, д. 19/8 Санкт-Петербург</span>
+      </div>
+    </section>
+    <section class="map">
+      <iframe
+        src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1998.6037861610246!2d30.3230474!3d59.9387165!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4696310fca145cc1%3A0x42b32648d8238007!2z0JHQvtC70YzRiNCw0Y8g0JrQvtC90Y7RiNC10L3QvdCw0Y8g0YPQuy4sIDE5LzgsINCh0LDQvdC60YIt0J_QtdGC0LXRgNCx0YPRgNCzLCAxOTExODY!5e0!3m2!1sru!2sru!4v1661197247635!5m2!1sru!2sru"
+        width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy"
+        referrerpolicy="no-referrer-when-downgrade"></iframe>"
+    </section>
+    <footer class="footer__">
```

Comment:
Что-то тут не так пошло с классом
```suggestion
    <footer class="footer">
```